modules/stanford_events_importer/stanford_events_importer.module

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * @file
+ * File description.
+ *
+ * Long description.
+ */
+
+use Drupal\migrate_tools\MigrateBatchExecutable;
+use Drupal\migrate\MigrateMessage;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\user\Entity\User;
+use Drupal\stanford_events_importer\StanfordEventsImporter;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Alter the config form to add the migrate_tools UI.
+ */
+function stanford_events_importer_form_config_pages_stanford_events_importer_form_alter(array &$form, FormStateInterface $form_state) {
+  $user = User::load(\Drupal::currentUser()->id());
+  $form['actions']['#type'] = 'fieldset';
+
+  $form['actions']['import'] = [
+    '#type' => 'submit',
+    '#value' => t('Save & Import'),
+    '#name' => 'op',
+    '#button_type' => "primary",
+    '#submit' => [
+      "::submitForm",
+      "::save",
+      "stanford_events_importer_form_config_pages_stanford_events_importer_form_alter_submit",
+    ],
+  ];
+
+  $form['actions']['update_opts'] = [
+    '#type' => 'submit',
+    '#value' => t('Update Org & Category Options'),
+    '#name' => 'op',
+    '#submit' => ["stanford_events_importer_update_opts"],
```

Comment:
you could use the service class function maybe

Reply:
I never made it a service class but rather just left it as a utility class. Not sure how often we might want to use it.